Text/PrettyPrint/ANSI/Leijen.hs

```diff
@@ -158,74 +160,5 @@ module Text.PrettyPrint.ANSI.Leijen (
    column, columns, nesting, width
    ) where
 
-import Text.PrettyPrint.ANSI.Leijen.Internal
-
-#if __GLASGOW_HASKELL__ >= 710
-import Data.Monoid ((<>))
-#elif __GLASGOW_HASKELL__ >= 704
-import Data.Monoid (Monoid, mappend, mconcat, mempty, (<>))
-#else
-import Data.Monoid (Monoid, mappend, mconcat, mempty)
-#endif
-
--- $DocumentAlgebra
```

Comment:
Did you mean to delete these Haddocks? `$DocumentAlgebra` is referenced in the export list.

Reply:
undone